lib/appoptics_apm/frameworks/rails/inst/action_controller3.rb

```diff
@@ -48,9 +44,10 @@ def process_action_with_appoptics(*args)
 
           process_action_without_appoptics(*args)
         rescue Exception
-          kvs[:Status] = 500
-          AppOpticsAPM::API.log(nil, 'info', kvs)
-          raise
+         kvs[:Status] = 500
```

Comment:
Did you mean to have 1 space indentation on this block?

Reply:
nope ;)